### DISCUSSION TITLE
Error generating changes

### DISCUSSION THREAD

**Discussion by ukraine-sdg** [open-sdg/open-sdg#2051] [Q&A] [answered]
We tried to make changes to the value of indicator 1.3.2. https://github.com/ukraine-sdg/data/actions. When we created the pull request: all checks failed. Previously, these changes were made successfully. Could tell me please where the error is. 
![image](https://github.com/open-sdg/open-sdg/assets/92239156/295f3302-7826-429d-ab48-4b677bb386c0)
![image](https://github.com/open-sdg/open-sdg/assets/92239156/db02a711-1bdd-4f90-a917-c80cb798e3d8)

**Comment by christoph-LSN** [comment, marked as answer]
@ukraine-sdg do you tried this: https://github.com/open-sdg/open-sdg/discussions/2020

**Comment by ukraine-sdg** [comment]
Made changes 
![image](https://github.com/open-sdg/open-sdg/assets/92239156/902c7356-cadd-4ced-9ac7-a5d7aba644bc)
but again errors
![image](https://github.com/open-sdg/open-sdg/assets/92239156/9222e631-6605-4edd-8896-81a6e2b5ef38)

**Comment by christoph-LSN** [comment]
@ukraine-sdg following your link to your requrements.txt there is not line with pydantic==1.*

https://github.com/ukraine-sdg/data/blob/develop/scripts/requirements.txt

**Comment by ukraine-sdg** [comment]
Thank you! I fix the error.
